osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/GetConfigurationObjectService.java

```diff
@@ -22,12 +24,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class GetConfigurationObjectService implements ProtocolService {
+public abstract class GetConfigurationObjectService extends AbstractConfigurationObjectService {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(GetConfigurationObjectService.class);
 
+  protected DlmsDeviceRepository dlmsDeviceRepository;
+
+  public GetConfigurationObjectService(final DlmsDeviceRepository dlmsDeviceRepository) {
```

Comment:
Abstract classes should not have public constructors. Constructors of abstract classes can only be called in constructors of their subclasses. So there is no point in making them public. The protected modifier should be enough.

Reply:
constructor is now protected